apps/explorer/lib/explorer/application.ex

```diff
@@ -39,6 +39,7 @@ defmodule Explorer.Application do
     # Children to start in all environments
     base_children = [
       Explorer.Repo,
+      Explorer.Repo.Replica1,
```

Comment:
I don't think we actually want this in the supervision tree

Reply:
Why not?

Reply:
I was under the impression we weren't going to use it, but if we are then ofc it makes sense.